src/main/scala/net/liftmodules/ng/ExecutionContextProvider.scala

```diff
@@ -0,0 +1,32 @@
+package net.liftmodules.ng
+
+import scala.concurrent.ExecutionContext
+
+object AngularExecutionContext { self =>
+  implicit var ec: ExecutionContext = ExecutionContext.global
+
+  def apply(ec: ExecutionContext) {
+    this.ec = ec
+  }
+
+  implicit val provider: ExecutionContextProvider = new ExecutionContextProvider {
+    override def ec: ExecutionContext = self.ec
+  }
+}
+
+trait ExecutionContextProvider {
+  def ec: ExecutionContext
+}
+
+object ExecutionContextProvider {
+  implicit class EnhancedExecutionContext(e: ExecutionContext) {
+    def asProvider: ExecutionContextProvider = new ExecutionContextProvider {
+      override def ec: ExecutionContext = e
```

Comment:
You're absolutely correct this implementation of `ExecutionContextProvider` does _NOT_ solve the serialization issue. My expectation is the majority of lift-ng users will not be using lift-cluster and just need an easy way for using the API. 

And after going through some iterations, I might even remove this `asProvider` conversion, now that I've settled on using `AngularExecutionContext` as commented on above.

Reply:
Awesome. The only thing I would suggest is to add a comment that says the same thing you explained here. Because people might mistakenly start using this even with clustering and wouldn't realize the problem until it was runnin.